### ISSUE TITLE
"auto-hide-size" doesn't respect "0" value

### ISSUE THREAD

**Issue by cazacugmihai** [mate-desktop/mate-panel#810]
#### Expected behaviour
When panel is in `auto-hide` mode and `auto-hide-size` is set to `0`, the panel should be completely hidden.

#### Actual behaviour
A part of the panel is still visible (having 1 pixel height).

#### Steps to reproduce the behaviour
dconf write /org/mate/panel/toplevels/toplevel-0/auto-hide true
dconf write /org/mate/panel/toplevels/toplevel-0/auto-hide-size 0

#### MATE general version
1.20.1

#### Package version
1.20.1-3ubuntu1

#### Linux Distribution
Ubuntu 18.04 LTS x86_64

**Comment by gustavolinux** [comment]
The same here

**Comment by raveit65** [comment]
Not sure, but is the panel accessible with auto-hide-size=0 ?
Maybe there is a good reason to have one pixel for expanding the panel?

**Comment by gustavolinux** [comment]
> Not sure, but is the panel accessible with auto-hide-size=0 ?
> Maybe there is a good reason to have one pixel for expanding the panel?

Yes, it works just like it was set to 1, but the panel line still appears when it's hidden

**Comment by raveit65** [comment]
> Yes, it works just like it was set to 1, but the panel line still appears when it's hidden

From where do you know this?
Or do you hope it works?

**Comment by gustavolinux** [comment]
> > Yes, it works just like it was set to 1, but the panel line still appears when it's hidden
> 
> From where do you know this?
> Or do you hope it works?

I'm using MATE on my main pc
